### PR TITLE
Do NOT Merge[RW-858][risk=no] updating workspace dashboard/about page

### PR DESCRIPTION
**Pull request by UrsaStutsman**
Replace this this template with your PR description.
Please remember to keep in mind the security levels outlined in [CONTRIBUTING.md](CONTRIBUTING.md) and to 
include a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title

* **no**: None 
* **low**: Low chance of potential impact to, or exposure of patient data
* **moderate**: Moderate chance of potential impact to, or exposure of patient data
* **severe**: Severe chance of potential impact to, or exposure of patient data

Please also:

* Get thumbs from reviewer(s)
* Verify all tests go green, including CI tests

### CODE REVIEW

File: ui/src/app/views/workspace/component.html [hunk 1]
```diff
@@ -1,103 +1,16 @@
...
-    <h2>{{workspace.name}}</h2>
-    <div style="display: flex; justify-content: space-between">
-      <div style="margin-left: -0.5rem;">
-        <button class="btn btn-link" [class.selected]="tabOpen === Tabs.Notebooks" (click)="tabOpen = Tabs.Notebooks">
-          Notebooks ({{notebookList.length}})
-        </button>
-        <button class="btn btn-link final" [class.selected]="tabOpen === Tabs.Cohorts" (click)="tabOpen = Tabs.Cohorts">
-          Cohorts ({{cohortList.length}})
-        </button>
+    <h2 style="height: 65px">{{greeting}}</h2>
```
Comment (UrsaStutsman) [inline ui/src/app/views/workspace/component.html]: Still need a decision on what to do if user does not have write permissions. Do we display cards greyed out, or not display at all? @lstrano-broad @fknabe

File: ui/src/app/styles/cards.css [hunk 2]
```diff
@@ -7,3 +7,20 @@
 .card * {
   margin-top: 0rem;
 }
+
+.create-card {
```
Comment (blrubenstein) [inline ui/src/app/styles/cards.css]: I'd make this `.card.create-card` to make a bit more specific? To ensure it overrides `.card`

File: ui/src/app/views/workspace/component.html [hunk 3]
```diff
@@ -1,103 +1,16 @@
...
+    <div class="card-area">
```
Comment (blrubenstein) [inline ui/src/app/views/workspace/component.html]: Until My PR is in, this is going to block access to review cohorts. I don't think we want to do that yet? (There's sorta some weird dependencies here...)

File: ui/src/app/views/workspace/component.html (hunk 1 above)
Comment (blrubenstein) [reply ui/src/app/views/workspace/component.html]: I suspect we want greyed out? But defer to Lou and Fritz. 

I think we want this header to the right of the create card area? Just to check?

Comment (UrsaStutsman) [reply ui/src/app/views/workspace/component.html]: From Insight it looks like that greeting is above the create card area

File: ui/src/app/views/workspace/component.ts [hunk 4]
```diff
@@ -90,6 +91,7 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
   cohortDescriptionComparator = new CohortDescriptionComparator();
```
Comment (UrsaStutsman) [inline ui/src/app/views/workspace/component.ts]: Not removing some of these as I'm not sure what may/may not still be in use for Bru's interdependent work

File: ui/src/app/views/workspace/component.spec.ts [hunk 5]
```diff
@@ -139,35 +139,35 @@ describe('WorkspaceComponent', () => {
...
-    workspacePage.readPageData();
-    expect(workspacePage.cohortsTableRows.length).toEqual(expectedCohorts);
-  }));
-
-  it('displays correct information when notebooks selected.', fakeAsync(() => {
-    workspacePage.readPageData();
-    tick();
-    expect(workspacePage.notebookTableRows.length).toEqual(1);
-  }));
+  // it('displays correct information when cohorts selected.', fakeAsync(() => {
```
Comment (UrsaStutsman) [inline ui/src/app/views/workspace/component.spec.ts]: Not removing these entirely until this page is finished, and we have appropriate tests to replace these (or have rewritten to work for new UI setup)

**Pull request closed without merging**